packages/atomic/src/components/result-template-components/atomic-result-multi-value-text/atomic-result-multi-value-text.tsx

```diff
@@ -50,6 +50,11 @@ export class AtomicResultMultiText {
    */
   @Prop() public maxValuesToDisplay = 3;
 
+  /**
+   * If the field isn't indexed as a multi value field, values are going to be split using this delimiter.
+   */
+  @Prop() public delimiter: string | null = null;
```

Comment:
Could the delimiter be purely CSS (::after & content)

Reply:
It's not added to the multi value text, it's used to split it.